gnomad_qc/v3/resources/sample_qc.py

```diff
@@ -73,55 +116,124 @@ def get_v3_relatedness_annotated_ht() -> Table:
+gnomad_v2_qc_sites = TableResource(
+    "gs://gnomad-public/resources/grch38/gnomad_v2_qc_sites_b38.ht"
+)
 
-# Dense MT of v3 samples at QC sites
-v3_qc = MatrixTableResource('gs://gnomad/sample_qc/mt/genomes_v3/gnomad_v3_qc_mt_v2_sites_dense.mt')
+# Dense MT of samples at QC sites
+qc = VersionedMatrixTableResource(
+    CURRENT_RELEASE,
+    {release: MatrixTableResource(f"gs://gnomad/sample_qc/mt/genomes_v{release}/gnomad_v{release}_qc_mt_v2_sites_dense.mt") for release in RELEASES}
+)
 
 # PC relate PCA scores
-v3_pc_relate_pca_scores = TableResource(f'{SAMPLE_QC_ROOT}/gnomad_v3_qc_mt_v2_sites_pc_scores.ht')
+pc_relate_pca_scores = VersionedTableResource(
+    CURRENT_RELEASE,
+    {release: TableResource(f"{get_sample_qc_root(release)}/gnomad_v{release}_qc_mt_v2_sites_pc_scores.ht") for release in RELEASES}
+)
 
 # PC relate results
-v3_relatedness = TableResource(f'{SAMPLE_QC_ROOT}/gnomad_v3_qc_mt_v2_sites_relatedness.ht')
+relatedness = VersionedTableResource(
+    CURRENT_RELEASE,
+    {release: TableResource(f"{get_sample_qc_root(release)}/gnomad_v{release}_qc_mt_v2_sites_relatedness.ht") for release in RELEASES}
+)
 
-# Sex imputation results
-v3_sex = TableResource(f'{SAMPLE_QC_ROOT}/gnomad_v3_sex.ht')
+# Sex imputation results	
+sex = VersionedTableResource(
+    CURRENT_RELEASE,
+    {release: TableResource(f"{get_sample_qc_root(release)}/gnomad_v{release}_sex.ht") for release in RELEASES}
+)
 
 # Samples to drop for PCA due to them being related
-pca_related_samples_to_drop = TableResource(f'{SAMPLE_QC_ROOT}/gnomad_v3_related_samples_to_drop_for_pca.ht')
+pca_related_samples_to_drop = VersionedTableResource(
+    CURRENT_RELEASE,
+    {release: TableResource(f"{get_sample_qc_root(release)}/gnomad_v{release}_related_samples_to_drop_for_pca.ht") for release in RELEASES}
+)
+
+# Related samples to drop for release	
+release_related_samples_to_drop = VersionedTableResource(
+    CURRENT_RELEASE,
+    {release: TableResource(f"{get_sample_qc_root(release)}/gnomad_v{release}_related_release_samples_to_drop.ht") for release in RELEASES}
+)
 
-# Related samples to drop for release
-release_related_samples_to_drop = TableResource(f'{SAMPLE_QC_ROOT}/gnomad_v3_related_release_samples_to_drop.ht')
+# Inferred sample populations	
+pop = VersionedTableResource(
+    CURRENT_RELEASE,
+    {release: TableResource(f"{get_sample_qc_root(release)}/gnomad_v{release}_pop.ht") for release in RELEASES}
+)
 
-# Inferred populations
-pop = TableResource(f'{SAMPLE_QC_ROOT}/gnomad_v3_pop.ht')
-pop_tsv_path = 'gs://gnomad/sample_qc/temp/genomes_v3/gnomad_v3_RF_pop_assignments.txt.gz'
-pop_rf_path = 'gs://gnomad/sample_qc/temp/genomes_v3/gnomad_v3_pop.RF_fit.pickle'
+
+def pop_tsv_path(version: str = CURRENT_RELEASE) -> str:
+    """
+    Path to tab delimited file indicating inferred sample populations
+
+    :param version: Version of sample QC path to return
```

Comment:
Is "sample QC" the correct description here? Same question for `pop_rf_path`.

Reply:
I think gnomAD version is more clear.